docs/advanced-usage/clerk-idp.mdx

```diff
@@ -84,20 +85,63 @@ The endpoint returns detailed token information such as if the token is still ac
 > [!WARNING]
 > This endpoint is protected. You must provide your Clerk Client ID and Client Secret credentials to authenticate.
 
-### OAuth 2.0 grant types
+### Grant types
 
-Clerk OAuth 2.0 provider supports the following flows and grant types:
+Clerk's OAuth 2.0 IdP implementation supports the following authorization flows and grant types:
 
 - [Authorization Code](https://oauth.net/2/grant-types/authorization-code/)
 - [PKCE](https://oauth.net/2/pkce/)
 - [Refresh token](https://oauth.net/2/grant-types/refresh-token/)
 
-## Frequently asked questions (FAQ)
+## OpenID Connect (OIDC)
 
-### When do the tokens expire?
+After a user successfully authenticates using the OIDC flow, they receive an ID token along with other tokens.
+
+The ID token is a JWT (JSON Web Token) that contains standard JWT claims as defined in RFC 7519, as well as additional custom claims that represent the authenticated user's profile information. The token is signed using your instance's private key and can be verified using the corresponding public key.
+
+{/* TODO: How do they use the public key to validate the token? */}
+
+> [!WARNING]
+> You MUST validate the ID token before using any of the user information that contains. To do that, you can use your instance's public key.
+
+The ID token includes the following standard claims:
+
+| Standard claim | Description |
+| - | - |
+| `iss` | The issuer of the token, which matches your Clerk Frontend API URL |
+| `sub` | The subject of the token, which matches the authenticated user ID |
+| `aud` | The intended audience of the token, which matches the used Client ID |
+| `exp` | The expiration time of the token |
+| `iat` | The time at which the token was issued |
+| `jti` | A unique identifier for the token |
+
+Depending on the granted scopes, the ID token can include the following additional claims:
+
+| Additional claim | Description |
+| - | - |
+| `nonce` | The nonce provided and used during the request |
+| `family_name` | The user's last name |
+| `given_name` | The user's first name |
+| `name` | The user's full name |
+| `picture` | The user's avatar URL |
+| `preferred_username` | The user's username |
+| `email` | The user's primary email address |
+| `email_verified` | Whether the user's primary email address is verified |
+| `public_metadata` | The user's public metadata |
+| `private_metadata` | The user's private metadata |
+| `unsafe_metadata` | The user's unsafe metadata |
+
+### OIDC prompt
 
-Authorization codes expire after 10 minutes. Access tokens expire after 2 hours. Refresh tokens expire after 3 days.
+{/* TODO: Where do they use the `prompt` parameter? */}
```

Comment:
While they are constructing their authorization request. Usually by using additional settings/props of the library of their choice